src/http/server/handlers/deflate_handler.cr

```diff
@@ -0,0 +1,33 @@
+require "zlib"
+
+class HTTP::DeflateHandler < HTTP::Handler
+  DEFLATE_TYPES = %w(text/html text/plain text/xml text/css text/javascript application/javascript)
```

Comment:
@bcardiff agree, defaults are good, but having the option to extend or replace such list, specifically the case of the missing `application/json` option.